src/context/DatastoreProvider.js

```diff
@@ -10,10 +10,11 @@ import { DatastoreReducer, initialState } from './DatastoreReducer';
 
 const axios = require('axios').default;
 
-const searchUrl = process.env.REACT_APP_LEGACY_RESULTS === 'true'
+const useLegacyResults = process.env.REACT_APP_LEGACY_RESULTS;
+const searchUrl = useLegacyResults === 'true'
```

Comment:
Might this be better as a one line like `const searchUrl = process.env.REACT_APP_LEGACY_RESULTS === 'true'`

Reply:
Disregard about the one line change. But do add it to `.env.example`.

Reply:
This actually isn't a new environment variable, it was added [here](https://github.com/amida-tech/saraswati-dashboard/pull/83/files#diff-a3046da0d15a27e89f2afe639b25748a7ad4d9290af3e7b1b6c1a5533c8f0a8c). 

Though I have a feeling you made sure to remind me to add it in that PR as well.